backend/solr/doc.py

```diff
@@ -66,12 +66,12 @@ def as_dict(self) -> dict:
         return {
             "id": self.id,
             "tags": self.tags,
-            "title": self.title,
-            "type": self.file_type,
-            "language": self.lang,
-            "size": self.size,
-            "creation_date": self.creation_date,
-            "content": self.content,
+            "title": self.title[0],
+            "type": self.file_type[0],
+            "language": self.lang[0],
+            "size": self.size[0],
+            "creation_date": self.creation_date[0],
+            "content": self.content[0],
```

Comment:
Are you really receiving arrays? Because I thought I changed that.

EDIT: Yeah this is definetely is breaking the tests. I am now sending you the correct data. So no need extract them out of the array